Library/Formula/xmlto.rb

```diff
@@ -40,10 +40,10 @@ def install
 +++ xmlto-0.0.25/xmlto.in
 @@ -209,7 +209,7 @@
  export VERBOSE
- 
+
  # Disable network entities
 -XSLTOPTS="$XSLTOPTS --nonet"
 +#XSLTOPTS="$XSLTOPTS --nonet"
- 
+
```

Comment:
XML validation still fails despite this patch when `xmlto` needs to download stuff from the internet. There seem to be other instances of hardcoded `nonet` options preventing `xmlto` from downloading the required files. See lines 545 and 548 [here](https://svn.fedorahosted.org/svn/xmlto/xmlto.in). I removed `--nonet` from those two lines and xml validation works now.